tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -100,11 +100,7 @@ def __init__(self, marionette):
         Wait(self.marionette).until(lambda m: url_bar.text != 'system.gaiamobile.org')
 
     def go_to_url(self, url):
-        # In private windows, the URL bar displayed is not the regular Rocket bar. But once you
-        # tap that URL bar, the regular Rocketbar comes back, in front of the private window.
-        # That's why we wait for the private window to be hidden
         self._root_element.find_element(*self._url_bar_locator).tap()
-        Wait(self.marionette).until(lambda m: self._root_element.get_attribute('aria-hidden') == 'true')
```

Comment:
Yes, from opening the (private) browser window, this attribute doesn't get 'hidden' the first time you tap on the url bar. The 2nd time it does.